generated/nirfmxnr/nirfmxnr.proto

```diff
@@ -1130,7 +1133,8 @@ enum NiRFmxNRInt32AttributeValues {
   NIRFMXNR_INT32_DOWNLINK_TEST_MODEL_MODULATION_TYPE_QAM16 = 2;
   NIRFMXNR_INT32_DOWNLINK_TEST_MODEL_MODULATION_TYPE_QAM64 = 3;
   NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE1 = 0;
-  NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE2 = 1;
+  NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE2_1 = 1;
```

Comment:
I'm tagging the PR because it's source breaking. Existing clients that use this enum will have to update their source when they regenerate the stubs with protoc. Existing clients that do not re-protoc will continue to work, sending numeric value `1` for `NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE2 `.

Since the header change has the same characteristics, I assume this is fine.